### PR TITLE
Feature/us 31 imp modify the response body of the post operation to login users

### PR DESCRIPTION
**Pull request by blancaagc**
This is the PR of the response of the post operation login

### CODE REVIEW

File: src/main/java/com/agilethought/atsceapi/dto/LoginDataResponse.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.agilethought.atsceapi.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class LoginDataResponse {
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/dto/LoginDataResponse.java:8]: I don't like name for this class. I would prefer "LoginResponse". But it is just a recommendation.

**Review by ivanflores15 — APPROVED** [review]

**Review by luisgalindoluna — APPROVED** [review]
Looks good. 
Some advice for you, please keep in mind how you choose to edit your files, I'm not sure what you did in the controller but it appears as the whole file was edited, which is not the case and in a project that is becoming increasingly bigger with each sprint these kind of slip-ups only make reviewing harder for us, the reviewers. So do keep that in mind for any other code changes you have to do.

**Comment by ivanflores15** [comment]
Same for me, appers to remove all class and create new one. But there is an option to avoid whitespaces for code review purposes. 
Most of the times it is because format code in intellij does not match with eclipse format. 

image

**Review by EmiliaGuillen-dev — APPROVED** [review]
approved

**Pull request merged**